confdgnmi/src/confd_gnmi_client.py

```diff
@@ -58,7 +59,9 @@ def get_capabilities(self) -> gnmi_pb2.CapabilityResponse:
         log.info("==>")
         request = gnmi_pb2.CapabilityRequest()
         log.debug("Calling stub.Capabilities")
+        log_rpc.debug("RPC - CapabilitiesRequest:\n%s", str(request))
```

Comment:
`str(request)` is always empty here

Reply:
that's right - i didn't want to specialize the code here in case "extensions" come into play later and/or underlying library changes